lib/src/executable/options.dart

```diff
@@ -373,7 +373,9 @@ class ExecutableOptions {
   Map<String, String> _listSourceDirectory(String source, String destination) {
     return {
       for (var path in listDir(source, recursive: true))
-        if (_isEntrypoint(path))
+        if (_isEntrypoint(path) &&
+            // Don't compile a CSS file to its own location.
+            (source != destination || p.extension(path) != '.css'))
```

Comment:
Total nit: might be clearer deMorgans'ed:

!(source == destination && p.extension(path) == '.css')

Reply:
Done.